doc/Extensions/Applications.md

```diff
@@ -229,6 +230,28 @@ The application should be auto-discovered as described at the top of
 the page. If it is not, please follow the steps set out under `SNMP
 Extend` heading top of page.
 
+# backupninja
+
+A small shell script that reports status of last backupninja backup.
+
+## SNMP Extend
+
+1: Download the [backupninja
+script](https://github.com/librenms/librenms-agent/blob/master/snmp/backupninja.py)
+to `/usr/local/src/backupninja.py` on your backuped server.
+```
+wget https://raw.githubusercontent.com/librenms/librenms-agent/master/snmp/backupninja.py -O /usr/local/src/backupninja.py`
```

Comment:
We tell people to install these scripts in /etc/snmp/ i.e: https://docs.librenms.org/Extensions/Applications/#apache